src/app/App.js

```diff
@@ -924,7 +940,7 @@ class App extends EventEmitter {
 			historyState.scrollTop = this.popstateScrollTop;
 			historyState.scrollLeft = this.popstateScrollLeft;
 		}
-		this.updateHistory_(title, redirectPath, nextScreen.beforeUpdateHistoryState(historyState), opt_replaceHistory);
```

Comment:
Sorry guys, that was from Mike's original PR. I though I removed that when solving the conflicts.